### PR TITLE
Updated code to compile on Xcode 8 beta 4

### PR DESCRIPTION
**Pull request by ThibaultKlein**
Please make sure I didn't break anything. I updated the pod sources and the example project by running with Xcode 8 beta 4.

### CODE REVIEW

File: Pod/Classes/Cards/Card.swift [hunk 1]
```diff
@@ -40,7 +40,7 @@ public struct Card {
      
      - returns: A card with the provided parameters.
      */
-    public static func create(number: String, cvc: String, expiry: String) throws -> Card {
+    public static func create(_ number: String, cvc: String, expiry: String) throws -> Card {
```
Comment (DannyVancura) [inline Pod/Classes/Cards/Card.swift]: Isn't it more descriptive and in line with the new Swift 3 style to keep this as is and call it like this: `Card.create(number: ..., cvc: ..., expiry: ...)`?

Comment (ThibaultKlein) [inline Pod/Classes/Cards/Card.swift]: Yes I made the update, although it doesn't require an update anywhere else

File: Pod/Classes/Cards/Expiry.swift [hunk 2]
```diff
@@ -144,18 +144,19 @@ extension Expiry: CustomStringConvertible {
  
  - returns: The date with month and year and time set to one minute before the following month.
  */
-private func toDate(month: UInt, year: UInt) -> Date? {
+private func toDate(_ month: UInt, year: UInt) -> Date? {
```
Comment (DannyVancura) [inline Pod/Classes/Cards/Expiry.swift]: Maybe we should rename this to `dateWith(month: UInt, year: UInt)`? To comply with this rule:

* [Prefer method and function names that make use sites form grammatical English phrases.](https://swift.org/documentation/api-design-guidelines/#strive-for-fluent-usage)

File: Pod/Classes/UI/CardTextField.swift [hunk 3]
```diff
@@ -588,24 +596,24 @@ public class CardTextField: UITextField, NumberInputTextFieldDelegate {
...
+            .flatMap({return $0.isFirstResponder})
+            .reduce(true, {$0 && $1}) {
             return false
         }
         return numberInputTextField.becomeFirstResponder()
     }
-    
-    public override func isFirstResponder() -> Bool {
+
+    public override func responds(to aSelector: Selector!) -> Bool {
```
Comment (DannyVancura) [inline Pod/Classes/UI/CardTextField.swift]: This should be `public override var isFirstResponder: Bool {`. Currently this causes a crash.

Comment (ThibaultKlein) [inline Pod/Classes/UI/CardTextField.swift]: Good catch, I'll make the update

File: Pod/Classes/Cards/Expiry.swift (hunk 2 above)
Comment (ThibaultKlein) [inline Pod/Classes/Cards/Expiry.swift]: I agree, updated

**Pull request closed without merging**